docs/guides/ultimate-guides/torrents/ultimate-intro-to-torrents/index.md

```diff
@@ -0,0 +1,152 @@
+
+## BitTorrent Basics
+
+BitTorrent decentralizes the file sharing process. 
+Rather than one server being required to serve a file to many users, which can become bottle necked, BitTorrent allows for many users to share files with each other by joining a "swarm".
+Users within the swarm can advertise they have a file, or a part of it, and exchange "pieces" with other users in the swarm.
+
+![BitTorrent Swarm](./img/tracker-light-mode.svg#gh-light-mode-only)![BitTorrent Swarm](./img/tracker-dark-mode.svg#gh-dark-mode-only)
+
+### Torrent files
+
+In order to distribute a file using BitTorrent, a ["torrent"](https://en.wikipedia.org/wiki/Torrent_file) file is created, which contains metadata about the file(s) to be shared.
+You can use a tool such as [TorrentEditor](https://torrenteditor.com/) to view the contents of a torrent file online. Here for example is the contents of the Ubuntu 23.04 Server ISO.
+
+:::info
+
+#### Ubuntu 23.04 Live Server Torrent Details
+
+**Filename:** ubuntu-23.04-live-server-amd64.iso.torrent
+
+##### Hash Details
+
+Info Hash: D6B4535B A8F2B340 12BC6335 69F113E7 7017E032
+
+##### Tracker Information
+
+**Tracker URL:** https://torrent.ubuntu.com/announce
+
+**Seeds:** 158, **Peers:** 4, **Completed:** 1
+
+Additional Tracker: https://ipv6.torrent.ubuntu.com/announce
+
+**Seeds:** 158, **Peers:** 4 **Completed:** 1
+
+##### Metadata
+
+- **Created On:** 04/20/23 12:14:49
+- **Created By:** mktorrent 1.1
+- **Comment:** Ubuntu CD releases.ubuntu.com
+- **Piece Length\*:** 256 KB
+- **Private\*:** Off - Unset
+
+##### Files
+
+**Filename:** ubuntu-23.04-live-server-amd64.iso
+
+**File Size:** 2.46 GB
+:::
+
+The torrent file itself is incredibly small, for this 2.46 GB ISO the torrent itself is only 200 KB.
+
+### Magnet links
+
+The data inside a torrent file is so small, that it can, in fact be can be represented as a URL rather than a file. 
+A ['Magnet link'](https://en.wikipedia.org/wiki/Magnet_URI_scheme) is a URL scheme that allows you to share information about a torrent as a link, rather than a file.
+This means websites that want to share torrents, no longer had to worry about serving files, making it extremely robust to simply share a _static_ webpage with links to all the files you want to share.
+Static webpages are extremely easy to host, robust against traffic spikes and take-down attempts, and can often even be hosted for free on a number of services online.
+This helped make torrent indexers like ThePirateBay _decentralized_. In 2012, in an effort to ensure the longevity of the content on ThePirateBay and thanks to their recent conversion to magnet links, the site was replicated by users and [reduced to a 90 MB torrent file](https://torrentfreak.com/the-pirate-bay-returns-121213/) that anyone could copy and re-host.
```

Comment:
[Proofreading]
```suggestion
This helped make torrent indexers like ThePirateBay _decentralized_. In 2012, in an effort to ensure the longevity of the content on ThePirateBay and thanks to their recent conversion to magnet links, Torrent Freak's site was replicated by users and [reduced to a 90 MB torrent file](https://torrentfreak.com/the-pirate-bay-returns-121213/) that anyone could copy and re-host.
```

Reply:
It was ThePirateBay that was replicated, should we use `their` site instead?

Reply:
> It was ThePirateBay that was replicated, should we use `their` site instead?

That could work! :)